### PR TITLE
Contact section #208

### PR DESCRIPTION
**Pull request by kire21**
- Fixed responsiveness for the contact section
- Created textArea reusable component 
- Used TextInput, TextArea and Button components in the contact form
- Made some changes for better responsive in textInput
- Changed file names to small letter cases
- The images for desktop and mobile on the bottom have 2px empty space and it is not good, the designer needs to fix this
- Deleted some code in the globals/boilerplate added from @Mikivitkovski  for auto field input, because it makes problems for the textInput

### CODE REVIEW

File: components/module-components/Contact/Contact.tsx [hunk 1]
```diff
@@ -1,35 +1,67 @@
...
-const handleCaptchaSuccess = () => {};
+// const handleCaptchaSuccess = () => {};
 
-const handleCaptchaError = () => {};
+// const handleCaptchaError = () => {};
 
-const Contact: React.FC = () => {
+const Contact = () => {
   const { theme } = useTheme();
+  const lightTheme = theme === 'light';
```
Comment (RevanchistX) [inline components/module-components/Contact/Contact.tsx:16]: boolean variable names should start with "is" => "isLightTheme"

**Comment by edichoska** [comment]
Screenshot 2024-05-17 at 12 33 22
Screenshot 2024-05-17 at 12 32 41
Screenshot 2024-05-17 at 12 10 46
General reponsiveness issues as shown on the images.

**Comment by edichoska** [comment]
Also fix validation problems.
Most critical is that it throws an error when you type space after entering first Name

**Comment by edichoska** [comment]
> @edichoska I fixed image responsiveness and form validation for full name (max 100 characters, accept Latin, Cyrillic characters, and space between first and last name and other things for check validations), email (I found the best practice for email validation and used that, but it also, in the end, can accept different characters I didn't see how to solve this), and message (min 100 characters, Latin and Cyrillic characters).

Thank you @kire21 for fixing the validation. Much better now.
I would shorten the requirement for the textarea to 20? 
Maybe someone just wants to ask a short question. They should be able to do that.
Also the email, now we can enter something like emi@m.c and it goes through

The image on mobile devices looks cut of from the sides.

**Comment by kire21** [comment]
> > @edichoska I fixed image responsiveness and form validation for full name (max 100 characters, accept Latin, Cyrillic characters, and space between first and last name and other things for check validations), email (I found the best practice for email validation and used that, but it also, in the end, can accept different characters I didn't see how to solve this), and message (min 100 characters, Latin and Cyrillic characters).
> 
> Thank you @kire21 for fixing the validation. Much better now. I would shorten the requirement for the textarea to 20? Maybe someone just wants to ask a short question. They should be able to do that. Also the email, now we can enter something like [emi@m.c](mailto:emi@m.c) and it goes through
> 
> The image on mobile devices looks cut of from the sides.

@edichoska I am done with the changes...

**Comment by edichoska** [comment]
Screenshot 2024-05-19 at 01 46 23
Error messages should be using error text color

**Comment by kire21** [comment]
> Screenshot 2024-05-19 at 01 46 23
> * Error messages should be using error text color
> * Придружи ни се button should lead to our discord(you can find the link in the Footer social media section)

If I use the error text color (red) for error messages, it is unreadable with very bad contrast... also the font-size is 12px, and many people have problems with their eyesight.

image

**Comment by kire21** [comment]
> > Screenshot 2024-05-19 at 01 46 23
> > * Error messages should be using error text color
> > * Придружи ни се button should lead to our discord(you can find the link in the Footer social media section)
> 
> If I use the error text color (red) for error messages, it is unreadable with very bad contrast... also the font-size is 12px, and many people have problems with their eyesight.
> 
> image

Or our audience will be only young people with sharp vision. :D

**Comment by edichoska** [comment]
> > > Screenshot 2024-05-19 at 01 46 23
> > > * Error messages should be using error text color
> > > * Придружи ни се button should lead to our discord(you can find the link in the Footer social media section)
> > 
> > 
> > If I use the error text color (red) for error messages, it is unreadable with very bad contrast... also the font-size is 12px, and many people have problems with their eyesight.
> > image
> 
> Or our audience will be only young people with sharp vision. :D

12-13-14px for error messages is pretty standard. 
I have communicated with the design team about the error color not having enough contrast with the dark background.
Lets just add it now so we can merge this. And when they are done we will change the color in colors.scss

**Comment by kire21** [comment]
@edichoska I am done.

**Review by edichoska — APPROVED** [review]
@kire21  Great work! Thank you for all the changes. 👏

**Pull request merged**